src/main/java/net/hypixel/nerdbot/command/ItemGenCommands.java

```diff
@@ -128,15 +128,18 @@ public void generateFullItem(GuildSlashEvent event,
         renderBackground = Objects.requireNonNullElse(renderBackground, true);
 
         // building the description for the item
-        BufferedImage generatedDescription = builder.buildItem(event, name, rarity, itemLore, type, disableRarityLinebreak, alpha, padding, maxLineLength, true);
-        if (generatedDescription == null) {
-            return;
+        BufferedImage generatedDescription = null;
+        if (itemName != null && rarity != null && itemLore != null) {
+            generatedDescription = builder.buildItem(event, itemName, rarity, itemLore, type, disableRarityLinebreak, alpha, padding, maxLineLength, true);
+            if (generatedDescription == null) {
+                return;
+            }
         }
 
         // building the item for the which is beside the description
         BufferedImage generatedItem = null;
-        if (itemName != null) {
-           generatedItem = builder.buildUnspecifiedItem(event, itemName, extraModifiers);
+        if (itemID != null) {
+           generatedItem = builder.buildUnspecifiedItem(event, itemID, extraModifiers);
             if (generatedItem == null) {
                 return;
```

Comment:
Wouldn't this just make the command fail? Should we not send a reply saying something went wrong?

Reply:
If someone decides to not provide a item id, then it doesn't render it to the image. But if a full image cannot be rendered, then it says it